pep-0646.rst

```diff
@@ -457,18 +484,27 @@ all arguments must be a ``Tuple`` parameterised with the same types.
     foo((0,), (1, 2))  # Error
     foo((0,), ('1',))  # Error
 
-Following `Type Variable Tuples Must Have Known Length`_, note
-that the following should *not* type-check as valid (even though it is, of
-course, valid at runtime):
+Using a Type Variable Tuple Inside a Tuple
+''''''''''''''''''''''''''''''''''''''''''
+
+Finally, by using a type variable tuple inside a tuple of other
+types, we can refer to prefixes or suffixes of the variadic
+argument list. For example:
 
 ::
 
-    def foo(*args: *Ts): ...
+    # os.execle takes arguments 'path, arg0, arg1, ..., env'
+    def execle(path: str, *args: *Tuple[*Ts, Mapping[str, str]]) → None: ...
+
+    # Note that this is different to
+    def execle(path: str, *args: *Ts, env: Mapping[str, str]) → None: ...
+    # which would make `env` a keyword-only argument.
```

Comment:
Again, I'd rather see less expository text in the comments here.

Reply:
Done.